deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -41,25 +34,33 @@ public abstract class OpenApiGeneratorCodeGenBase implements CodeGenProvider {
 
     private static final String DEFAULT_PACKAGE = "org.openapi.quarkus";
 
+    // It will be ignored if INPUT_BASE_DIR is specified
     @Override
     public String inputDirectory() {
         return "openapi";
     }
 
+    @Override
+    public boolean shouldRun(Path sourceDir, Config config) {
+        return Files.isDirectory(config.getOptionalValue(INPUT_BASE_DIR, Path.class).orElse(sourceDir), new LinkOption[0]);
```

Comment:
Why not just:

```suggestion
        return Files.isDirectory(config.getOptionalValue(INPUT_BASE_DIR, Path.class).orElse(sourceDir));
```

?

Reply:
because the original shouldRun method in CodeGenProvider had a new LinkOption[0] and wasn't sure about it 😅

Reply:
because the original shouldRun method in CodeGenProvider had a new LinkOption[0] and wasn't sure about it 😅